app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/components/ConversionHostsList.js

```diff
@@ -1,17 +1,61 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Grid, ListView, Toolbar } from 'patternfly-react';
+import ListViewToolbar from '../../../../common/ListViewToolbar/ListViewToolbar';
 
 const ConversionHostsList = ({ conversionHosts }) => (
-  <React.Fragment>
-    <h2>TODO: render a list view here using the ListViewToolbar pattern we have elsewhere</h2>
-    {conversionHosts.map(conversionHost => (
-      <pre key={conversionHost.id}>{JSON.stringify(conversionHost)}</pre>
-    ))}
-  </React.Fragment>
+  <ListViewToolbar
+    filterTypes={ConversionHostsList.filterTypes}
+    sortFields={ConversionHostsList.sortFields}
+    listItems={conversionHosts}
+  >
+    {({
+      filteredSortedPaginatedListItems,
+      renderFilterControls,
+      renderSortControls,
+      renderActiveFilters,
+      renderPaginationRow
+    }) => (
+      <React.Fragment>
+        <Grid.Row>
+          <Toolbar>
+            {renderFilterControls()}
+            {renderSortControls()}
+            {renderActiveFilters(filteredSortedPaginatedListItems)}
+          </Toolbar>
+        </Grid.Row>
+        <div style={{ overflow: 'auto', paddingBottom: 300, height: '100%' }}>
```

Comment:
This is outside the scope of this PR, because this pattern is all over the place, but I think at some point we should probably move all of these inline `style` props into CSS, it's a little messy that we use both. It's fine for now though.

Reply:
Yep, I definitelly agree with moving this into CSS (we use it in 3 other places as well).